plugins/modules/aci_l3out_logical_interface_profile_eigrp_profile.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Jason Juenger (@jasonjuenger) <jasonjuenger@gmail.com>
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_logical_interface_profile_eigrp_profile
+short_description: Manage Layer 3 Outside (L3Out) EIGRP interface profile (eigrp:IfP)
+description:
+- Manage L3Out logical interface profile EIGRP profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ interface_profile_name, logical_interface ]
+  eigrp_profile:
+    description:
+    - Name of the EIGRP interface profile to create.
+    type: str
+    aliases: [ name, eigrp_profile_name ]
```

Comment:
dow e need this or can we use the interface profile name for it, like done in ospf module?

Reply:
updated to match the ospf module as requested, the eigrpIfP name is now the same as the interface profile name.